src/test/OpenEXRTest/testOptimizedInterleavePatterns.cpp

```diff
@@ -577,13 +581,16 @@ test (Schema writeScheme, Schema readScheme, bool nonfatal, bool tiny)
     cout << left << setw (53) << q.str ();
 
     FrameBuffer writeFrameBuf;
-    Box2i       dw = writefile (writeScheme, writeFrameBuf, tiny);
+    // only allow NaN and infinity vales if file is read and written as half float
+    // (otherwise casting between half and float may cause different bit patterns)
+    bool allowNonFinite = (writeScheme._types == nullptr && readScheme._types==nullptr);
```

Comment:
Trivial nit, but the other uses of this term have lowercase 'f': `allowNonfinite`.